app/controllers/conversations_controller.rb

```diff
@@ -1,2 +1,49 @@
 class ConversationsController < ApplicationController
+
+  def new
+    @convo = Conversation.new
+  end
+
+  def show
+    @convo = Conversation.find(params["id"])
+  end
+
+  def index
+    @page = params[:page].to_i
+    per_page = 15
```

Comment:
We want to display conversations in the last 24 hours, shouldn't have an arbitrary 15